Octokit/Models/Request/PullRequestReviewSubmit.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Diagnostics;
+using System.Globalization;
+using Octokit.Internal;
+using System.Collections.Generic;
+
+namespace Octokit
+{
+    /// <summary>
+    /// Used to filter requests for lists of pull requests.
+    /// </summary>
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    public class PullRequestReviewSubmit : RequestParameters
+    {
+        public PullRequestReviewSubmit()
+        {
+        }
+        
+        /// <summary>
+        /// Filter pulls by head user and branch name in the format of "user:ref-name".
+        /// </summary>
+        public string Body { get; set; }
+
+        /// <summary>
+        /// Filter pulls by base branch name.
+        /// </summary>
+        public PullRequestReviewSubmitEvents Event { get; set; }
+                
+        internal string DebuggerDisplay
+        {
+            get
+            {
+                return string.Format(CultureInfo.InvariantCulture, "Event: {0} ", Event);
+            }
+        }
+    }
+
+    public enum PullRequestReviewSubmitEvents
+    {
+        [Parameter(Value = "APPROVE")]
+        APPROVE,
```

Comment:
Use camel case for the enum members (but keep the `[Parameter]` tags in place to specify the actual Api values

eg

``` csharp
public enum PullRequestReviewSubmitEvents
{
[Parameter(Value = "APPROVE")]
Approve,

[Parameter(Value = "REQUEST_CHANGES")]
RequestChanges,

[Parameter(Value = "COMMENT")]
Comment
}
```

Also perhaps rename the enum since it can be used for both Submit and Create actions